app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,30 +1,15 @@
-import React, { useState } from "react";
+import React from "react";
 
 import { Alert } from "neetoui";
 
-const DeleteAlert = ({ onClose }) => {
-  const [deleting, setDeleting] = useState(false);
-
-  const handleDelete = async () => {
-    try {
-      setDeleting(true);
-      onClose();
-    } catch (error) {
-      logger.error(error);
-      setDeleting(false);
-    }
-  };
-
-  return (
-    <Alert
-      isOpen
-      isSubmitting={deleting}
-      message="Are you sure you want to continue? This cannot be undone."
-      title="Delete note?"
-      onClose={onClose}
-      onSubmit={handleDelete}
-    />
-  );
-};
+const DeleteAlert = ({ onClose, handleDelete }) => (
+  <Alert
+    isOpen
+    message="Are you sure you want to continue? This cannot be undone."
```

Comment:
`continue` with what? The alert messages should be explicit. Here we performing a specific irreversible action. Thus the messages should be updated accordingly.

Reply:
fixed with the following message Done ✅ 

> Are you sure about deleting this note? This can't be undone, Press continue to delete permanently.